app/services/search_indexer.rb

```diff
@@ -243,6 +243,7 @@ def self.index(obj, force: false)
     end
 
     if Category === obj && (obj.saved_change_to_name? || force)
+      SearchIndexer.queue_category_posts_reindex(obj.id)
```

Comment:
this is new ... but I guess it is safe ... was only done as a side effect before.